aodndata/aims/mmp_ctd.py

```diff
@@ -26,8 +26,7 @@ def preprocess(self):
     def dest_path(filepath):
         nc_obj = Dataset(filepath, mode='r')
         year = datetime.strptime(nc_obj.time_coverage_start, '%Y-%m-%dT%H:%M:%SZ').strftime("%Y")
-        site_short_name = nc_obj.site_short_name
 
         return os.path.join('AIMS', 'Marine_Monitoring_Program', 'CTD_profiles',
-                            year, site_short_name, '{filename}'.
+                            year, '{filename}'.
                             format(filename=os.path.basename(filepath)))
```

Comment:
The `''.format()` does nothing here. Why not just...
```suggestion
                            year, os.path.basename(filepath))
```